code/game/objects/effects/misc.dm

```diff
@@ -92,4 +92,36 @@
 	name = "horrific experiment"
 	desc = "Some sort of pod filled with blood and vicerea. You swear you can see it moving..."
 	icon = 'icons/obj/cloning.dmi'
-	icon_state = "pod_g"
\ No newline at end of file
+	icon_state = "pod_g"
+
+
+//Makes a tile fully lit no matter what
+/obj/effect/fullbright
+	icon = 'icons/effects/alphacolors.dmi'
+	icon_state = "white"
+	plane = LIGHTING_PLANE
+	layer = LIGHTING_LAYER
+	blend_mode = BLEND_ADD
+
+
+/obj/effect/dummy/lighting_obj
+	name = "lighting fx obj"
+	desc = "Tell a coder if you're seeing this."
+	icon_state = "nothing"
+	light_color = "#FFFFFF"
+	light_range = MINIMUM_USEFUL_LIGHT_RANGE
+	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
+
+/obj/effect/dummy/lighting_obj/Initialize(mapload, _color, _range, _power, _duration)
+	. = ..()
+	set_light(_range ? _range : light_range, _power ? _power : light_power, _color ? _color : light_color)
+	if(_duration)
+		QDEL_IN(src, _duration)
+
+/obj/effect/dummy/lighting_obj/moblight
+	name = "mob lighting fx"
+
+/obj/effect/dummy/lighting_obj/moblight/Initialize(mapload, _color, _range, _power, _duration)
+	. = ..()
+	if(!ismob(loc))
+		return INITIALIZE_HINT_QDEL
```

Comment:
Is this working now? I know you said you had issue with it before?

Reply:
For some reason this one QDEL's fine, but some things don't properly trigger this. Maybe has to do with references.